src/WebJobs.Script/CompositeTraceWriter.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    internal sealed class CompositeTraceWriter : TraceWriter
```

Comment:
Add a TODO comment on here that this is a dupe of fx class until it becomes public (if it does).